content/docs/reference/cd-google-cloud-build.md

```diff
@@ -68,6 +70,16 @@ steps:
   - 'BUILD_TYPE=$_BUILD_TYPE'
 ```
 
+In the above configuration, we use a substitution `_BUILD_TYPE` to indicate to the build environment about the type of build it is running. This allows us to run a `preview` or an `update` based on the build type. To configure the substitution, you need to setup build triggers for each type of branch builds you intend to support for your project.
```

Comment:
I think we should make it _even_ more clear that both substitutions are provided by the build trigger. Something to the effect of:

> In the configuration above, both `_BUILD_TYPE` and `_INSECURE_SUBSTITUTION_PULUMI_ACCESS_TOKEN` are configured as a part of the Cloud Build Trigger that runs the Cloud Build. You can read more about triggers at ...

Reply:
Done.